src/components/Form/FormFileInput.react.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import * as React from "react";
+import cn from "classnames";
+
+type Props = {|
+  +className?: string,
+  +value?: string | number | boolean,
+  +name?: string,
+  +label?: string,
+  +disabled?: boolean,
+  +readOnly?: boolean,
+  +onChange?: (event: SyntheticInputEvent<HTMLInputElement>) => void,
+|};
+
+function FormFileInput({
```

Comment:
Should this have state to track the file or will we let the html component take care of that?

Reply:
It's now using state for displaying the filename but otherwise yeah html component, or controlled with onChange